src/artoo.beep.js

```diff
@@ -26,8 +26,22 @@
   }
 
   // Playing the base64 sound
-  artoo.beep = function(sound) {
-    var chosenSound;
+  artoo.beep = function(a1, a2) {
+    var sound,
+        callback,
+        chosenSound;
+
+    if (typeof a1 === 'function') {
+      callback = a1;
+    }
+    else {
+      sound = a1;
+      if (typeof a2 === 'function') 
+        callback = a2;
+      else
+        throw Error('artoo.beep: second argument have to be a function.');
```

Comment:
The `beep` method cannot be called without callback. This is problematic, my liege.